src/url-utils.js

```diff
@@ -52,3 +52,16 @@ export function repoBaseUrl(repo, gateway = ipfsDefaultConf.gateway) {
     gateway
   )
 }
+
+const GITHUB = 'https://github.com'
+const GITLAB = 'https://gitlab.com'
+const BITBUCKET = 'https://bitbucket.com'
+
+export const parseHub = url =>
```

Comment:
Maybe this would be better named as `sanitizeCodeRepositoryUrl()`? `parseHub()` is quite vague from a glance.